### PR TITLE
AppParameters test

### PR DESCRIPTION
**Pull request by Valentine1898**

### CODE REVIEW

File: packages/router/src/grpc/view-protocol-server/app-parameters.test.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    slashingPenaltyMisbehavior: '10000000',
+    slashingPenaltyDowntime: '10000',
+    signedBlocksWindowLen: '10000',
+    missedBlocksMaximum: '9500',
+  },
+  feeParams: {},
+  distributionsParams: {
+    stakingIssuancePerBlock: '1',
+  },
+});
```
Comment (grod220) [inline packages/router/src/grpc/view-protocol-server/app-parameters.test.ts:80]: `fromJson()` unfortunately does not give type hints if the object shape changes. Let's go with this:

```typescript

const testData = new AppParameters({
  chainParams: {
    chainId: 'penumbra-testnet-titan',
    epochDuration: 719n,
  },
  communityPoolParams: {
    communityPoolSpendProposalsEnabled: true,
  },
  governanceParams: {
    proposalVotingBlocks: 17280n,
    proposalDepositAmount: {
      lo: 10000000n,
    },
    proposalValidQuorum: '40/100',
    proposalPassThreshold: '50/100',
    proposalSlashThreshold: '80/100',
  },
  ibcParams: {
    ibcEnabled: true,
    inboundIcs20TransfersEnabled: true,
    outboundIcs20TransfersEnabled: true,
  },
  stakeParams: {
    unbondingEpochs: 2n,
    activeValidatorLimit: 80n,
    baseRewardRate: 30000n,
    slashingPenaltyMisbehavior: 10000000n,
    slashingPenaltyDowntime: 10000n,
    signedBlocksWindowLen: 10000n,
    missedBlocksMaximum: 9500n,
  },
  feeParams: {},
  distributionsParams: {
    stakingIssuancePerBlock: 1n,
  },
});

```

**Review by grod220 — APPROVED** [review]
One change and then free to merge 👍

File: packages/router/src/grpc/view-protocol-server/app-parameters.test.ts (hunk 1 above)
Comment (erwanor) [reply packages/router/src/grpc/view-protocol-server/app-parameters.test.ts:80]: Heads up that we are removing chain params today, the chain id will be a top-level field of the application parameters and the epoch duration moves to shielded pool parameters. This should be the last churn of public interfaces for a while. xref https://github.com/penumbra-zone/penumbra/pull/3703

Comment (Valentine1898) [reply packages/router/src/grpc/view-protocol-server/app-parameters.test.ts]: added TODO about removing `chainParams`

**Pull request merged**